frontend/src/components/LevelMissionInfoBanner/LevelMissionInfoBanner.css

```diff
@@ -38,11 +43,22 @@
 	align-items: center;
 	box-sizing: border-box;
 	height: auto;
-	padding: 0 0.625rem;
+	padding: 0 0.5rem;
 	border-radius: 50%;
 	background-color: var(--action-button-text-colour);
 	color: var(--action-button-background-colour);
 	font: 1rem serif;
 	font-weight: bold;
 	aspect-ratio: 1;
 }
+
+@media (width < 62.5rem) {
+	.level-mission-info-banner .banner-button-container {
+		flex-direction: column;
+		gap: 0.125rem;
```

Comment:
I think this can stay at 0.5rem, seems little to gain from squishing the buttons together.